en/docs/Learn/Extensions/SAML2SSO/configuring-identity-server-as-idp-for-sso.md

```diff
@@ -34,7 +34,7 @@ For example, let's share a JDBC user store (MySQL) with both the WSO2 Identity S
 
 3.  Create a MySQL database (e.g., 410\_um\_db) and run the `<API-M_HOME>/dbscripts/mysql.sql` script on it to create the required tables.
 
-4.  Open the `<API-M_HOME>/repository/conf/datasources/master-datasources.xml` and `<API-M_HOME>/repository/conf/user-mgt.xml` file and check the datasource configuration for the database that you use for the shared user store and user management information. Go to `<API-M_HOME>/repository/conf/deployment.toml` file and add configurations. For example, you can share a single user store as follows. If you are sharing multiple datasources, you need to define a datasource for each of the user stores that you are working with, so that they can be shared.
+4.  Go to `<API-M_HOME>/repository/conf/deployment.toml` file and add database configurations. For example, you can share a single user store as follows. (If you are sharing multiple datasources, you need to define a datasource for each of the user stores that you are working with, so that they can be shared.)
```

Comment:
Add "the" before `<API-M_HOME>/repository/conf/deployment.toml` file.

Reply:
Thanks. fixed in https://github.com/wso2/docs-apim/pull/603/commits/00819e0f9b8510051be2641ff645f74f71966156